.github/workflows/black.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will install Python dependencies and run flake8.
+
+name: Python black checks
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-16.04
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.8]
```

Comment:
Why does the Python version differ for this workflow from the others?

Reply:
I thought black would complain if I ran it on multiple versions, I'll see if I can run it on multiple versions though.

Reply:
It complains, and also seems unnecessary, since it's just a code style check.

Reply:
The complaining looks to be due to the python version in the pre-commit hook being fixed on 3.8, which is something I see you just fixed. I agree on the unnecessary part though --- we can probably just run it on 3.x and be done with it.